app/components/finance/npq/contract_course_row.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Finance
+  module NPQ
+    class ContractCourseRow < BaseComponent
+      include FinanceHelper
+
+    private
+
+      delegate :course_identifier, :recruitment_target, :per_participant, to: :@npq_contract
+
+      def initialize(contract_course_row:)
```

Comment:
that's a bit confusing, looks like we're passing an instance of an `npq_contract` not a `contract_course_row`
```suggestion
      def initialize(npq_contract:)
```

Reply:
Yeah thats originally what I had it as, but these contract components require you to have the argument as the same name as the class, its weird.